storage-bigtable/src/lib.rs

```diff
@@ -855,6 +855,10 @@ impl LedgerStorage {
             .connection
             .put_protobuf_cells_with_retry::<generated::ConfirmedBlock>("blocks", &blocks_cells)
             .await?;
+        datapoint_info!(
```

Comment:
This may be pretty noisy for multi-warehouse-node metrics. Debug-level instead?

Reply:
Yeah, this effectively doubles the logs for that information. I was hoping you'd ask, because I would prefer to delete the log message that follows it. What would you think of that? In my opinion, this information is necessary in order to know whether the warehouse nodes are healthy, and so it's not really debug information.

Reply:
@joeaba , do you have an opinion about this?

Reply:
My new proposal is to replace the log line with the metric line; best of both worlds. You end up with something like this:

```
[2022-06-21T14:39:32.279931220Z INFO  solana_metrics::metrics] datapoint: storage-bigtable-upload-block slot=138440539i transactions=2093i bytes=1516622i
```

Reply:
A metric line makes more sense. I think I vote info, I'd want to have it by default on all the warehouse nodes I run